l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -2,27 +2,33 @@
 
 pragma solidity 0.8.20;
 
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
 import {BeaconProxy} from "@openzeppelin/contracts/proxy/beacon/BeaconProxy.sol";
 import {UpgradeableBeacon} from "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import "hardhat/console.sol";
 
 import {IL1ERC20Bridge} from "./interfaces/IL1ERC20Bridge.sol";
 import {IL2SharedBridge} from "./interfaces/IL2SharedBridge.sol";
+import {IL2SharedBridgeLegacy} from "./interfaces/IL2SharedBridgeLegacy.sol";
 import {IL2StandardAsset} from "./interfaces/IL2StandardAsset.sol";
+import {IL2StandardToken} from "./interfaces/IL2StandardToken.sol";
 import {IL2StandardDeployer} from "./interfaces/IL2StandardDeployer.sol";
 
 import {L2StandardERC20} from "./L2StandardERC20.sol";
 import {AddressAliasHelper} from "../vendor/AddressAliasHelper.sol";
 import {L2ContractHelper, DEPLOYER_SYSTEM_CONTRACT, IContractDeployer} from "../L2ContractHelper.sol";
 import {SystemContractsCaller} from "../SystemContractsCaller.sol";
 
+import {NATIVE_TOKEN_VAULT_VIRTUAL_ADDRESS} from "../common/Config.sol";
+
 /// @author Matter Labs
 /// @custom:security-contact security@matterlabs.dev
 /// @notice The "default" bridge implementation for the ERC20 tokens. Note, that it does not
 /// support any custom token logic, i.e. rebase tokens' functionality is not supported.
-contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable {
+contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, OwnableUpgradeable {
```

Comment:
We're setting ERA_CHAIN_ID and general naming is around L2

I have a feeling we should change the framework to focus on Hyperchain naming and assume that this contract would be deployed on all Hyperchains so we can bridged wrapped versions between them

Reply:
Also, I think good design would be to:

-  deploy 1 native vault on each chain (for native tokens)
-  deploy 1 standard token vault (which burns / mints wrapped tokens, which are deployed on native chain)

This would solve the native token vault question (not sure about native base token though)

Reply:
We are already assuming it is deployed to multiple hyperchains. We need Era_chain_id for legacy reasons in current upgrade. 

> Also, I think good design would be to:

What is the difference between NTV and STV?